### PR TITLE
Closes issue #4567

### PR DESCRIPTION
**Pull request by nilootpal**
`xmlns={name?.attributes?.xmlns}`
`viewBox={name?.attributes?.viewBox}`
Added optional chaining in Icon.jsx

Closing issue #4567

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2447,7 +2447,7 @@ See https://6.docs.plone.org/volto/upgrade-guide/index.html for more information
 - Use Plone logo @ericof
 - Update favicon and related tags with best practices @sneridagh
 - Enable to be able to use the internal proxy in production as well @sneridagh
-- Add runtime configuration for `@babel/plugin-transform-react-jsx` set to `automatic`. This enables the new JSX runtime: https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html So no longer `import React from 'react'` is needed anymore. @sneridagh
+- - Add runtime configuration for `@babel/plugin-transform-react-jsx` set to `automatic`. This enables the new JSX runtime: https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html So no longer `import React from 'react'` is needed anymore. @sneridagh
```
Comment (stevepiercy) [inline CHANGELOG.md:2450]: Please revert this change. You almost never should edit this file.

File: news/4569.documentation [hunk 2]
```diff
@@ -1 +0,0 @@
-Fix broken links at ReactJS.org. @stevepiercy
```
Comment (stevepiercy) [inline news/4569.documentation:1]: Do not delete this file. Please revert.

File: news/4567.internal [hunk 3]
```diff
@@ -0,0 +1 @@
+change conditional checking to optional chaining src/components/theme/Icon/Icon.jsx
```
Comment (stevepiercy) [inline news/4567.internal:1]: Give yourself credit.

```suggestion
Change conditional checking to optional chaining for a theme icon @nilootpal

```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
See review comments.

**Comment by stevepiercy** [comment]
@nilootpal

# STOP!

Why are you editing all these random files that have nothing to do with the issue that you are trying to fix?

**Comment by nilootpal** [comment]
Sorry @stevepiercy for changing such files. But the errors in the checks suggested so I did that. I think that my local repository was outdated than the branch, that is why this happened. Now I have reverted all those changes

File: CHANGELOG.md (hunk 1 above)
Comment (nilootpal) [reply CHANGELOG.md]: I have reverted the changes

File: news/4569.documentation (hunk 2 above)
Comment (nilootpal) [reply news/4569.documentation]: I have reverted the changes

**Comment by stevepiercy** [comment]
Correct. Keep in mind that Volto is under continual, rapid development. That means you must frequently merge `master` into your branch and push to the PR, or click the button in the GitHub UI to update the PR then pull down the changes to your branch.

Next step is for a maintainer to review your PR. Tip: check [git blame](https://github.com/plone/volto/blame/86cc15fe42b6df3d0a70d13149588aed00b6802f/src/components/theme/Icon/Icon.jsx) for the file's history to see who edited it in the past.

**Comment by nilootpal** [comment]
@sneridagh can you review the PR

**Review by sneridagh — APPROVED** [review]
